src/components/Modal/QRCode.svelte

```diff
@@ -0,0 +1,76 @@
+<script lang="typescript">
+    import { qrcode } from "pure-svg-code";
+    import Modal from "@app/Modal.svelte";
+    import Metamask from "@app/Metamask.svelte";
+    import { state } from '@app/session';
+    import type { Config } from '@app/config';
+    
+
+
+    export let uri: string;
+    export let config: Config;
+    export let caption = "Connect Wallet";
+
+
+    $: svgString = qrcode({
+      content: uri,
+      width: 225,
+      height: 225,
+      color: "black",
+      background: "white",
+      ecl: "M"
+    });
+
+    const onClickConnect = () => {
+
+      console.log(config, "from svelete");
+    
+      state.connectMetamask(config);
+};
```

Comment:
Indentation here is wrong -- but why doesn't the eslint "indent" rule catch this :roll_eyes:

Reply:
The new commit you made to package.json fixed this

Reply:
Looks like it isn't fixed..

Reply:
The whole file content is somehow wrongly indented